security_overview/network_security.sp

```diff
@@ -0,0 +1,48 @@
+variable "allowed_ips" {
+  type        = list(string)
+  default     = ["0."]
+  description = "A list of IPs allowed in Snowflake network policies."
+}
+
+benchmark "security_overview_network_security" {
+  title         = "Network Security"
+  description   = "Network security or isolation provides the first line of defense for Snowflake account."
+  documentation = file("./security_overview/docs/network_security.md")
+  children = [
+    control.security_overview_network_security_use_network_policies,
```

Comment:
As discussed earlier, can you please:
- Add another control that checks if all desired blocked IPs (which will be in a new variable, `blocked_ips`) are in the blocked list
- Update this control name to `control.security_overview_network_security_network_policy_allowed_list_set`